content/scripts/build.js

```diff
@@ -455,8 +481,9 @@ class Builder {
       !this.options.regenerateAllTitles
     ) {
       // XXX maybe this should become a Map instance.
-      this.allTitles = JSON.parse(
-        fs.readFileSync(allTitlesJsonFilepath, "utf8")
+      Object.assign(
```

Comment:
Why the copy?

Reply:
Ah! You explained this on Zoom. But the new task is the *explain* why this needs to be done.